src/main/java/com/zerobase/cafekiosk/beverage/service/BeverageServiceImpl.java

```diff
@@ -62,4 +62,21 @@ public void delete(Long id) {
 
     beverageRepository.deleteById(id);
   }
+
+  @Override
+  public Page<BeverageDto> frontBeverages(Long categoryId, Pageable pageable) {
+
+    Page<Beverage> beverages = beverageRepository.findAllByCategoryId(categoryId, pageable);
+
+    return beverages.map(BeverageDto::of);
```

Comment:
```suggestion
    return beverageRepository.findAllByCategoryId(categoryId, pageable).map(BeverageDto::of);
```
불필요한 선언은 하지 않아도 될것 같습니다